refinery/factory_boy/utils.py

```diff
@@ -212,3 +212,362 @@ def create_tool_with_necessary_models(tool_type, user=None):
     if user is not None:
         tool.set_owner(user)
     return tool
+
+
+def create_hg_19_data_set(user=None):
+    # Generally mocks the hg_19 local data set's study, assay, nodes,
```

Comment:
Maybe adding in tests to assert against the results of the parsed versions of these respective datasets would be good since these structures being correct are important to this new functionality